### PR TITLE
Setting up Godot with Rider on Windows

### PR DESCRIPTION
**Pull request by van800**

### CODE REVIEW

File: getting_started/scripting/c_sharp/c_sharp_basics.rst [hunk 1]
```diff
@@ -47,6 +47,21 @@ you can download just the
 instead.
 Make sure you at least have the .NET Framework 4.5 targeting pack installed, you can get it using any of the installers mentioned above inside the "Individual components" tab.
 
+Windows (JetBrains Rider)
+~~~~~~~
```
Comment (Calinou) [inline getting_started/scripting/c_sharp/c_sharp_basics.rst:51]: Number of tildes should match the heading. Also, add a blank line after the heading:

```suggestion
~~~~~~~~~~~~~~~~~~~~~~~~~

```

File: getting_started/scripting/c_sharp/c_sharp_basics.rst [hunk 2]
```diff
@@ -47,6 +47,21 @@ you can download just the
...
+
+- In Godot:
+
+   - Mono External Editor to JetBrains Rider
+   - Mono Build Tool to JetBrains Mono.
+
+- In Rider:
+ 
+   - TargetFrameworkVersion in csproj should match installed .NET Framework Developer Pack.
+   - Install `Godot support` plugin
```
Comment (Calinou) [inline getting_started/scripting/c_sharp/c_sharp_basics.rst:63]: Use bold when referencing UI elements:

```suggestion
   - Install **Godot support** plugin
```

File: getting_started/scripting/c_sharp/c_sharp_basics.rst [hunk 3]
```diff
@@ -47,6 +47,22 @@ you can download just the
 instead.
 Make sure you at least have the .NET Framework 4.5 targeting pack installed, you can get it using any of the installers mentioned above inside the "Individual components" tab.
 
+Windows (JetBrains Rider)
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+JetBrains Rider comes with bundled MSBuild, so you just need recent .NET Framework Developer Pack.
```
Comment (mhilbrunner) [inline getting_started/scripting/c_sharp/c_sharp_basics.rst:53]: Would add an article here: *... so you just **a** need recent .NET Framework Developer Pack.*

Comment (van800) [reply getting_started/scripting/c_sharp/c_sharp_basics.rst]: added it, thanks

**Comment by mhilbrunner** [comment]
Looks good now! Can you squash the commits into one please? :)

**Comment by van800** [comment]
Done!

**Comment by akien-mga** [comment]
@mhilbrunner Note that on godot-docs it's OK to use "Squash and Merge" for PRs.

**Comment by akien-mga** [comment]
Thanks!

**Comment by mhilbrunner** [comment]
Ah right :) Thanks again for the contribution!

**Pull request merged**